ProjectName/src/features/quickPay/QuickPay.tsx

```diff
@@ -1,50 +1,37 @@
 import React from 'react';
-import {
-  Text,
-  StyleSheet,
-  SafeAreaView,
-  View,
-  Pressable,
-  FlatList,
-} from 'react-native';
+import {Text, SafeAreaView, View, Pressable, FlatList} from 'react-native';
 import ListAccounts from './components/ListAccounts';
-
 import AddAcounts from './components/AddAcounts';
 import QuickPayStyle from './QuickPayStyle';
-const QuickPay = ({payAccounts}) => {
-
+import {Avatar} from '../../types/Avatar';
+const QuickPay = ({payAccounts}: {payAccounts: Avatar[]}) => {
```

Comment:
Is Avatar a suitable name for the pay accounts type? :/ 

Also please add a return type for functional components in all the different functional component files. 

Try adding https://github.com/typescript-eslint/typescript-eslint TS linter, it will help to see all the possible errors related to TS in your project.